packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -72,14 +72,14 @@ function SelectionTool(props: Props) {
         return;
       }
 
-      const { worldPt, sourceEvent } = evt;
+      const { worldPt } = evt;
       const dataEndPoint = worldToData(boundPointToFOV(worldPt, camera));
       setEndPoint(dataEndPoint);
 
       const worldEndPoint = dataToWorld(dataEndPoint);
       const worldStartPoint = dataToWorld(startPoint);
 
-      if (onSelectionChange && shouldInteract(sourceEvent)) {
+      if (onSelectionChange && isModifierKeyPressed) {
```

Comment:
`shouldInteract` checks that the `sourceEvent.button` is equal to the interaction button (in this case `MouseButton.Left = 0`). However, for pointer move events, [`sourceEvent.button` is equal to `-1`](https://www.w3.org/TR/pointerevents/#the-button-property) when the button didn't change which makes the check fail. 

The quick fix consists in checking only the modifier key instead of the key and the button (strategy already used in `Pan`).